client/signup/steps/p2-site/index.jsx

```diff
@@ -1,637 +0,0 @@
-import config from '@automattic/calypso-config';
-import { FormLabel, Gridicon } from '@automattic/components';
-import { getLanguage } from '@automattic/i18n-utils';
-import { createRef } from '@wordpress/element';
-import { reusableBlock, Icon } from '@wordpress/icons';
-import debugFactory from 'debug';
-import { localize } from 'i18n-calypso';
-import { includes, isEmpty, map, deburr, get, debounce } from 'lodash';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import FormButton from 'calypso/components/forms/form-button';
-import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
-import FormTextInput from 'calypso/components/forms/form-text-input';
-import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
-import formState from 'calypso/lib/form-state';
-import { getLocaleSlug } from 'calypso/lib/i18n-utils';
-import { logToLogstash } from 'calypso/lib/logstash';
-import { login } from 'calypso/lib/paths';
-import wpcom from 'calypso/lib/wp';
-import P2StepWrapper from 'calypso/signup/p2-step-wrapper';
-import { getValueFromProgressStore } from 'calypso/signup/utils';
```

Comment:
We should be able to remove the `getValueFromProgressStore` utility now, too.

Reply:
Done in 6146bfca079